tests/third_party/scipy/test_bessel.py

```diff
@@ -0,0 +1,33 @@
+import unittest 
+
+import jax._src.test_util as jtu
+import jax.numpy as jnp
+from jax.scipy.special import bessel_j0, bessel_j1, bessel_jn
+
+from jax.config import config
+from scipy.special import jv
+config.update("jax_enable_x64", True) # this is *absolutely essential* for the jax bessel function to be numerically stable
```

Comment:
If this is the case, then the implementations should raise errors if x64 is not enabled.

Is there any hope of computing bessel functions in float32?

Reply:
Yeah, they're not actually that bad in float32, but they won't make allclose with the scipy ones. The existing jax implementation straight up won't work in float32 because the series doesn't converge. I would suggest it doesn't need to raise an error, but we could comment in the docstrings that float64 is best.

Reply:
Specifically, it's not that bad for j0 and j1, but it goes bad if you try and calculate higher order jn with the recurrence relation in the vicinity of 0.